src/visual.ts

```diff
@@ -58,25 +61,16 @@ import VisualConstructorOptions = powerbi.extensibility.visual.VisualConstructor
 import ILocalizationManager = powerbi.extensibility.ILocalizationManager;
 import ISelectionManager = powerbi.extensibility.ISelectionManager;
 
-import {
-    IColorArray,
-    IMargin,
-    TableHeatMapChartData,
-    TableHeatMapDataPoint,
-} from "./dataInterfaces";
-
-import {
-    SettingsModel,
-    colorbrewer
-} from "./settings";
-
+import { IColorArray, IMargin, TableHeatMapChartData, TableHeatMapDataPoint } from "./dataInterfaces";
+import { SettingsModel, colorbrewer } from "./settings";
 import { FormattingSettingsService } from "powerbi-visuals-utils-formattingmodel";
-
-import {
-    ITooltipServiceWrapper,
-    TooltipEnabledDataPoint,
-    createTooltipServiceWrapper
-} from "powerbi-visuals-utils-tooltiputils";
+import IInteractivityService = interactivityBaseService.IInteractivityService;
+import { interactivitySelectionService, interactivityBaseService } from "powerbi-visuals-utils-interactivityutils";
+import { IInteractiveBehavior } from "powerbi-visuals-utils-interactivityutils/lib/interactivityBaseService";
```

Comment:
Same as previous. Direct imports are not the best option

Reply:
Same question, I can do it I'm just curious as to what benefit it would provide?